dags/sync_dist.py

```diff
@@ -1,3 +1,4 @@
+import os
```

Comment:
I believe we don't need it any more as we move it to the `af_xxx` instead, do you mind to help for a cleanup?